ansible/roles/sysprep/tasks/debian.yml

```diff
@@ -32,16 +32,14 @@
       - /etc/apt/sources.list.d
     patterns: '*.list.disabled'
   register: repo_files
-  when:
-    disable_public_repos | default(false) | bool
-    and reenable_public_repos | default(true) | bool
+  when: (disable_public_repos | default(false) | bool and reenable_public_repos | default(true) | bool) or
```

Comment:
😺 we should revisit to simplify this in next PR.

Reply:
tbh these vars are not used anywhere else and are not documented. We could remove it in the future but didn't want to do that here.